docs/en/ingest-management/processors/processors.asciidoc

```diff
@@ -0,0 +1,137 @@
+[[elastic-agent-processor-configuration]]
+= Define processors
+
+//REVIEWERS: I think we need to define "processors." Here's my first shot.
+//I think we also need to make it clear that these are different from the
+//processors defined in ingest pipelines. Should we qualify the name?
+
+Processors are lightweight processing components that you can use to parse,
+filter, transform, and enrich data at the source. For example, you can use
+processors to:
+
+* reduce the number of exported fields
+* enhance events with additional metadata
+* perform additional processing and decoding
+* sanitize data
+
+Each processor receives an event, applies a defined action to the event, and
+returns the event. If you define a list of processors, they are executed in the
+order they are defined.
+
+[source,yaml]
+----
+event -> processor 1 -> event1 -> processor 2 -> event2 ...
+----
+
+[discrete]
+[[where-valid]]
+== Where are processors valid?
+
+The processors described in this section are valid:
+
+* **Under integration settings in the Integrations UI in {kib}**. For example,
+when configuring an Nginx integration, you can define processors for a specific
+dataset under **Advanced options**. The processor in this example adds geo
+metadata to the Nginx access logs collected by {agent}:
++
+[role="screenshot"]
+image::images/add-processor.png[Screen showing how to add a processor to an integration policy]
++
+NOTE: Some integrations do not currently support processors.
+
+* **Under input configuration settings for standalone {agent}s**. For example:
++
+[source,yaml]
+----
+inputs:
+  - type: logfile
+    use_output: default
+    data_stream:
+      namespace: default
+    streams:
+      - data_stream:
+          dataset: nginx.access
+          type: logs
+        ignore_older: 72h
+        paths:
+          - /var/log/nginx/access.log*
+        tags:
+          - nginx-access
+        exclude_files:
+          - .gz$
+        processors:
+          - add_host_metadata:
+              cache.ttl: 5m
+              geo:
+                name: nyc-dc1-rack1
+                location: '40.7128, -74.0060'
+                continent_name: North America
+                country_iso_code: US
+                region_name: New York
+                region_iso_code: NY
+                city_name: New York
+          - add_locale: null
+----
+
+//REVIEWERS: Does this config look correct? I modified the policy created
```

Comment:
Need input from reviewers on this question.

Reply:
LGTM